### PR TITLE
Regenerate confirmation token method

### PR DESCRIPTION
**Pull request by lee-kagiso**

### CODE REVIEW

**Comment by lee-kagiso** [comment]
@RiaanWest you can upgrade the cms to use this lib at your leisure

will let you create a new confirmation token for resending the confirmation mail

File: auth_backend/models.py [hunk 1]
```diff
@@ -64,18 +65,28 @@ def confirm_email(self, confirmation_token):
...
 
-        if not status == 200:
+        if not status == http.HTTP_200_OK:
             raise CASUnexpectedStatusCode(status, data)
 
         self.confirmation_token = None
         self.email_confirmed = timezone.now()
         self.save()
 
+    def regenerate_confirmation_token(self):
```
Comment (lee-kagiso) [inline auth_backend/models.py]: @RiaanWest this is the ticket

File: auth_backend/http.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+HTTP_200_OK = 200
+HTTP_201_CREATED = 201
+HTTP_204_NO_CONTENT = 204
+HTTP_404_NOT_FOUND = 404
+HTTP_409_CONFLICT = 409
+HTTP_422_UNPROCESSABLE_ENTITY = 422
+HTTP_500_INTERNAL_SERVER_ERROR = 500
```
Comment (colinfrankb) [inline auth_backend/http.py]: There's no need to do this as the requests library already has a long list of codes
eg.

`requests.codes.not_found`

Comment (lee-kagiso) [inline auth_backend/http.py]: yeah, i saw that, but they dont include the error code in their naming, and it is just a list of constants

Comment (colinfrankb) [inline auth_backend/http.py]: theirs has alternative names as well ... 
`407: ('proxy_authentication_required', 'proxy_auth', 'proxy_authentication'),`

anyway.

**Pull request merged**